collector/filesystem_bsd.go

```diff
@@ -33,6 +33,7 @@ import "C"
 
 const (
 	defIgnoredMountPoints = "^/(dev)($|/)"
+	MNT_RDONLY            = 0x1
```

Comment:
I have seen Go code (I think even in the standard library) that uses C convention if it helps mirroring OS constants. So it _might_ be OK to do it here.